dispatcher/config.ini

```diff
@@ -72,6 +72,14 @@ MasterDAQConfig = {
         }
     }
 
+VMEConfig = {
+  "0": "192.168.131.60",
+  "1": "192.168.131.61",
+  "2": "192.168.131.62",
+  "3": "192.168.131.63",
+  "4": "192.168.131.64"
+}
+
```

Comment:
Why are the IPs different from https://github.com/XENONnT/daqnt/pull/33/files#diff-2dac8abc3e3c1e09e1e08cfc16f6954cc245484ea396708142dc1f29d6be07f5R31-R34

Reply:
These are the addresses of the 5 VME crates. I'm not sure why you're comparing the production addresses to testing addresses.

Reply:
oops 🤦